packages/story-editor/src/app/media/utils/useFFmpeg.js

```diff
@@ -279,6 +279,66 @@ function useFFmpeg() {
     [getFFmpegInstance]
   );
 
+  /**
+   * Trim Video using FFmpeg.
+   *
+   * @param {File} file Original video file object.
+   * @param {string} start Time stamp of start time of new video. Example '00:01:00'.
```

Comment:
@barklund Mark here, that ffmpeg expects this format. This is what we should store in post meta.

Reply:
ffmpeg supports milliseconds, e.g. `00:01:02.345`. We want to support that too in the editor.

Reply:
Didn't say we didn't support milliseconds here. Just flagging the format is not a number, see original comment.
https://github.com/google/web-stories-wp/issues/8891#issuecomment-911737499

Reply:
Let's update the JSDoc to make that clear.

```suggestion
   * @param {string} start Time stamp of start time of new video. Example '00:01:02.345'.
```